package.json

```diff
@@ -26,7 +26,7 @@
         "elm-format": "0.8.7",
         "elm-optimize-level-2": "0.3.5",
         "elm-pages": "^3.0.19",
-        "elm-review": "2.11.1",
+        "elm-review": "2.12.0",
```

Comment:
From what I can tell, `>= 2.12` of the `elm-review` CLI is needed for "analyzing arbitrary files" (so using `withExtraFilesProjectVisitor`) https://github.com/jfmengels/node-elm-review/releases/tag/v2.12.0
I tried `2.13` version of the CLI, but that seemed to require `>= 2.15` of the library, so I just opted for the minimal upgrade to support the changes here.